documentation/modules/auxiliary/scanner/http/gavazzi_em_login_loot.md

```diff
@@ -0,0 +1,31 @@
+This module scans for Carlo Gavazzi Energy Meters login portals, performs a login brute force attack, will gather device firmware version and attempt to extract SMTP configuration. A valid, admin privileged user is required to extract SMTP password. In some older versions, SMTP config can be retrieved without any authentication. The module also exploits an access control vulnerability which allows an unauthenticated user to remotely dump EWplant.db database file - this db file contains information such as power/energy utilization data, tariffs, and revenue statistics. Vulnerable firmware versions - VMU-C EM prior to firmware Version A11_U05 and VMU-C PV prior to firmware Version A17.
```

Comment:
This module scans for Carlo Gavazzi Energy Meters login portals, performs a login brute force attack, **enumerates** device firmware version,  and attempt to extract **the** SMTP configuration. A valid, admin privileged user is required to extract **the** SMTP password. In some older **firmware** versions, **the** SMTP config can be retrieved without any authentication. 
**(newline)** The module also exploits an access control vulnerability which allows an unauthenticated user to remotely dump **the** database file **EWplant.db  .** **T**his db file contains information such as power/energy utilization data, tariffs, and revenue statistics. 
**(newline)** Vulnerable firmware versions **include:**
* VMU-C EM prior to firmware Version A11_U05
* VMU-C PV prior to firmware Version A17.